Specification.md

```diff
@@ -0,0 +1,87 @@
+- [Shared mime Database](https://www.freedesktop.org/wiki/Specifications/shared-mime-info-spec/)
+- [Startup Notification](https://www.freedesktop.org/wiki/Specifications/startup-notification-spec/)
+- [Trashcan support](https://www.freedesktop.org/wiki/Specifications/trash-spec/)
+- [Bookmark Interchange Format](https://pyxml.sourceforge.net/topics/xbel/)
+
+TBD: Not sure on these:
+- [MIME Applications Associations](https://www.freedesktop.org/wiki/Specifications/mime-apps-spec/)
+- [XSettings](https://www.freedesktop.org/wiki/Specifications/xsettings-spec/)
+- [Cursor Conventions](https://www.freedesktop.org/wiki/Specifications/cursor-spec/)
+- [Sound Theme](https://www.freedesktop.org/wiki/Specifications/sound-theme-spec/)
+
+### Window decorations
+
+Applications which are part of the xapp project should support server side window decorations and use them by default. Xapps are free to optionally as well support client side decorations [^1].
+
+### Menus
+
+If menus are required for a xapps, traditional menu bars should be used by default. It is fine to optionally support e.g. hamburger menus [^2].
+
+TBD: Do we want to force a default on this?
+
+### Theming
+
+User theming has to be supported by xapps (TODO: Where is the spec for general theming?) [^3]
+
+### Desktop Environment Agnostic
+
+If the same windowing, theme and icon theme is used, XApps are intended to be visually consistent across Linux desktop environments. As such, they should not have dependencies to desktop-specific styles.
+
+### Identification as xapp application
+
+If an application fulfills all above criteria and is supposed to be added to the xapp project, it has to add a paragraph to a README.md file located on the top level of its git project with the following text:
+
+> ## Xapp compliance
+>
+> This application follows the standards defined in the [Xapp specification v1.0.0](insert link here) and as such will run fine on linux desktop environments which support freedesktop.org standards. The UI of this application will be well integrated on desktop environments which support the xapp standard.
+
+The xapp project provides a set of [project-properties](https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-organization-settings/managing-custom-properties-for-repositories-in-your-organization) which allows to filter xapps for supported operating systems, the used license, the used toolkit and supported display servers. Please set these project-properties for your project accordingly.  
+
+### Repository Location
+
+Once an application is approved as an xapp application, it is either hosted directly by the Xapp project (so that the native issue tracker, git repository and infrastructure is used) or its repository is mirrored to the xapp project. On choosing the latter, the xapp project page will hide the possibility to file issues and show a link to the original repository location.
```

Comment:
This feels a little ambitious; what's the rationale behind encouraging projects to use Xapp hosting (where is that?), or even mirroring it?  Wouldn't Xapp just hosting a list of conformant projects be sufficient, and low overhead for all concerned?

Reply:
Ideally, there should be a differentiation between apps that will be (or want to be) developed under the xapp-project umbrella, and other apps that aren't necessarily under the xapp-project umbrella but do follow the XApp standards which we say are xapp-compliant applications. I believe we should clarify this differentiation and that the repository location requirements only apply to applications that want to be developed/supported/maintained under the xapp-projects umbrella.

Reply:
True, a filterable list of project names and links which do fulfill the xapp spec would as well be sufficient for the spec alone.

Afaik the idea of @clefebvre was to host some existing linux mint xapps outside of the linux mint project, to make clear that they are supposed to be used by other DE's as well .. in addition, there are plans to fork some of the gnome apps, like e.g. "gnome-scan", revert them to some previous state and maintain them here as a shared effort between DE's supporting xapps. (Here the [related Mint Blogpost](https://blog.linuxmint.com/?p=4675))

Foreign repositories hosting xapps (e.g. several Xfce applications) could then be mirrored to the xapp-project in order to be shown alongside the native ones, in order to get the same attention as these (I suppose that would be the only advantage of mirroring over just maintaining a list)

Reply:
We'll need a list no matter what, at least on the website. And if it's easy to automate github mirroring let's do it.

I think different people look at different things. Casual users might pay more attention to a website, devs might not even look at it but they might browse the list of repositories on xapp-project.

Being present in both places gives more visibility to the app and makes it clearer it's on board with the project.

Reply:
If for any reason we need two different levels we can show that in the list, and with badges on the READMEs, once it's mirrored it's visible anyway, it shouldn't really matter whether an app is primarily hosted on the XApp github or somewhere else imo.

Reply:
Just played a bit with Github custom project properties, I suppose they would be superior to badges on the README, because like that, it would be possible to filter for apps which fulfill specific criteria. [Here a minimal filter example](https://github.com/orgs/test-alexxcons-organization/repositories?q=props.Windowing%3AWayland). [Here all props of the project](https://github.com/test-alexxcons-organization/testrepo/settings/custom-properties).

E.g. if I want to know which XApps do support Wayland, having a possibility to filter will get important.

However, for sure we could as well implement a filter on the website ... maybe both?